metadata-models/src/main/pegasus/com/linkedin/dataset/FullTableBatchSpec.pdl

```diff
@@ -0,0 +1,8 @@
+namespace com.linkedin.dataset
+
+/**
+ * A "batch" indicating the entire table
+ */
+record FullTableBatchSpec {
```

Comment:
Why do we need all of these models? I think I may want @treff7es to provide review as well around the partitioning related models -- we have been thinking about modeling partitions as a first class concepts elsewhere in our model